docs/content/develop/step-4.md

```diff
@@ -9,6 +9,31 @@ menu = "develop"
 tags = [ "develop" ]
 +++
 
+We've been publishing our PatternFly Elements to the [@patternfly organization](https://www.npmjs.com/org/patternfly) on npm.
 
+To roll a new release, use the following steps:
 
-We've been publishing our PatternFly Elements to the [@patternfly organization](https://www.npmjs.com/org/patternfly) on npm.
+1. Start in the root directory of the patternfly-elements project.
+2. Check out the master branch and pull down the latest: `git checkout master && git pull`.
+3. To have lerna bump the verions, run: `npm run lerna version -- --no-git-tag-version --no-push --preid prerelease`.
+    - Choose the appropriate version bump type for the release you're publishing:
+        - if bumping a prerelease version (example: from 1.0.0-prerelease.2 to 1.0.0-prerelease.3), choose *Custom Prerelease*
+4. Assign that version to this variable for use below: `$NEW_VERSION: 1.0.0-prerelease.3`.
+5. Create a new branch: `git checkout -b release/$NEW_VERSION` (example: release/1.0.0-prerelease.3).
+5. Run a fresh install, build all the assets, and then force add those assets for the tag release:
+    - `npm install && npm run build && git add elements/*/*.{js,map,css} -f`
+6. Commit the updates: `git commit -am "$NEW_VERSION"`
+7. Roll the tag: `git tag "v$NEW_VERSION"`
+8. Remove the bundle files:
+    - `for e in elements/*; do git rm -f elements/$(basename $e)/$(basename $e).{js,css,map}; done`
```

Comment:
@mwcz This doesn't quite work for us anymore since some of our assets don't have the same name as the base element - do you have a suggested update for this string?

Reply:
That for loop you sent me in chat doesn't run on my system either -- it doesn't escape so it doesn't execute 🤷‍♂️ Could you edit this and put in what you think we should use here?

Reply:
Can you try this? `for e in elements/*; do find $e -maxdepth 1 \( -not -name "gulpfile.js" -not -name "rollup.config.js" \) \( -name "*.js" -or -name "*.css" -or -name "*.map" \) -exec git rm -f {} \; ;done`

So ugly, but it works well, at least on Linux.  Very curious if it works on Macs.